src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/If_V1Compat.txt

```diff
@@ -0,0 +1,13 @@
+#SETUP: PowerFxV1CompatibilityRules
+
+>> If(1>0, {a:1}, {b:2})
```

Comment:
nit - for documentation purposes, can you add a comment about what V1 does differently?